modules/tinymce/src/themes/silver/test/ts/browser/sidebar/SidebarShowOptionTest.ts

```diff
@@ -67,12 +71,13 @@ describe('browser.tinymce.themes.silver.sidebar.SidebarShowOptionTest', () => {
     McEditor.remove(editor);
   });
 
-  it('TINY-8710: Show a different sidebar on init', async () => {
+  it('TINY-8710: Sidebar name should be lowercased', async () => {
     const editor = await McEditor.pFromSettings<Editor>({
       ...settingsFactory(store),
-      sidebar_show: 'sidebartwo'
+      sidebar_show: 'SideBarTwo'
```

Comment:
If you're changing this, then shouldn't the name in the `addSidebar` call be changed as well? As the intention here is that they should match.